union-ai-admin/aws/script/generate.py

```diff
@@ -377,6 +377,17 @@ def create_updater_policy(role_type):
                         ),
                     ],
                 ),
+                Statement(
+                    Effect=Allow,
+                    Action=[
+                        Action("cloudfront", "UpdateCloudFrontOriginAccessIdentity"),
+                    ],
+                    Resource=[
+                        Sub(
+                            "arn:aws:cloudfront::${AWS::AccountId}:origin-access-identity/*"
+                        )
+                    ],
+                ),
```

Comment:
We don't need cloudfront in the dataplane, can we easily remove whatever is creating this from Opta instead of adding more permissions?